wallet/src/Pos/Wallet/Web/Methods/Misc.hs

```diff
@@ -192,6 +194,9 @@ instance MimeRender OctetStream WalletStateSnapshot where
 instance Buildable WalletStateSnapshot where
     build _ = "<wallet-state-snapshot>"
 
+instance Default WalletStateSnapshot where
+    def = WalletStateSnapshot { wssWalletStorage = def }
```

Comment:
Why do we need a `Default` instance for `WalletStateSnapshot`?

Reply:
We don't need it anymore. The first idea was to use it to create a `Arbitrary WalletStateSnapshot` from it (similar as you suggested today https://input-output-rnd.slack.com/archives/C819S481Y/p1519120198000571?thread_ts=1519115554.000460&cid=C819S481Y )